examples/solidity/coverage/single.sol

```diff
@@ -1,14 +1,14 @@
 contract C {
```

Comment:
I think echidna is having trouble with this example. Is it because of the `require` that cause a revert?

Reply:
I think the problem is both `mutateCall` and `shrinkTx` modify all the different args at once. We need something that mutates/shrinks less aggressively